### PR TITLE
p2-b6-r5 Paris

### PR DESCRIPTION
**Pull request by GrantSchreib**

### CODE REVIEW

File: bHY6yh-d41894/cod-bxxqqq_bHY6yh-d41894.xml [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+            <cb ed="#B" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d41894"><!-- p2l6-r5 -->
+            <head xml:id="bHY6yh-d41894-Hd1e108">Rubrica 5</head>
+            <head xml:id="bHY6yh-d41894-Hd1e111" type="question-title">De tyrannide et tyranno</head>
+            <p xml:id="bHY6yh-d41894-d1e114">
+              De <sic>tyra<lb ed="#B" break="no"/>nide</sic> 
+              et tyranno Rubrica 5
```
Comment (jeffreycwitt) [inline bHY6yh-d41894/cod-bxxqqq_bHY6yh-d41894.xml:65]: are we missing a line break here?

File: bHY6yh-d41894/cod-bxxqqq_bHY6yh-d41894.xml [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+              <lb ed="#B"/>nullus enim pusillamius insidiabitur 
+              <lb ed="#B"/>Secundum autem ut discredant in<lb ed="#B" break="no"/>uicem. 
+              non dissoluitur prius enim ty<lb ed="#B" break="no"/>rannus 
+              antequam credant subditi sibi 
+              <lb ed="#B"/>ipsis propter quod et epikeis aduersatur 
+              <lb ed="#B"/>tanquam nociuis ad principatum non so<lb ed="#B" break="no"/>lum 
+              propterea quod non uolunt subii dis<lb ed="#B" break="no"/>potice 
+              <pb ed="#B" n="70-v"/>
+              <cb ed="#B" n="a"/>
+              dispotita in serui sed et quia
```
Comment (jeffreycwitt) [inline bHY6yh-d41894/cod-bxxqqq_bHY6yh-d41894.xml:153]: missing line break?

File: bHY6yh-d41894/cod-bxxqqq_bHY6yh-d41894.xml [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+              <lb ed="#B"/>quam id quid in amicitia adiungitur</quote>
+            </p>
+            <p xml:id="bHY6yh-d41894-d1e342">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d41894-Rd1e370">Tvlius de amicitia</ref> <quote xml:id="bHY6yh-d41894-Qd1e346">In vita 
+              <lb ed="#B"/>tyrannorum nulla fides nul<lb ed="#B" break="no"/>la 
+              caritas nulla stabilis be<lb ed="#B" break="no"/>niuolentie 
+              potest esse fiducia omnia 
+              <lb ed="#B"/>semper suspecta atque solicita 
+              <lb ed="#B"/>nullus locus amicitie Qui enim 
+              eum aut diligit quem metuit
```
Comment (jeffreycwitt) [inline bHY6yh-d41894/cod-bxxqqq_bHY6yh-d41894.xml:192]: missing line break?

File: bHY6yh-d41894/cod-bxxqqq_bHY6yh-d41894.xml [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+            </p>
+            <p xml:id="bHY6yh-d41894-d1e363">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d41894-Rd1e393">SEneca tragediarum in<!--line jumps up--> <unclear>trahad</unclear><!--line jumps down--> capitulo 4</ref> <quote xml:id="bHY6yh-d41894-Qd1e367">vi<lb ed="#B" break="no"/>olenta
+              nemo imperia continuit 
+              <lb ed="#B"/>diu</quote>
+            </p>
+            <p xml:id="bHY6yh-d41894-d1e373">
+              ¶ <ref xml:id="bHY6yh-d41894-Rd1e404">In medea capitulo 4</ref> <quote xml:id="bHY6yh-d41894-Qd1e375">Iniqua nun<lb ed="#B" break="no"/>quam 
+              <cb ed="#B" n="b"/>
+              regna perpetuo manent</quote>
```
Comment (jeffreycwitt) [inline bHY6yh-d41894/cod-bxxqqq_bHY6yh-d41894.xml:205]: missing line break?

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]
I think we are missing a few line break here

**Comment by GrantSchreib** [comment]
corrected

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**